.github/workflows/ci.yaml

```diff
@@ -15,6 +15,8 @@ jobs:
     - name: Get Changed Terraform directories
       id: search
       uses: mozilla/tf-actions/matrixify@main
+      with:
+        ignore_dir: "**/examples"
```

Comment:
@cmharlow It seems like this works - you can look at my latest commits and see that once I added these lines linting passes. I then intentionally caused a formatting issue and the CI still fails as expected. If there's a better spot/convention for this ignore directory, let me know!